lms/templates/main_django.html

```diff
@@ -28,14 +28,14 @@
   <div class="window-wrap" dir="${static.dir_rtl()}">
     <a class="nav-skip" href="{% block nav_skip %}#content{% endblock %}">{% trans "Skip to main content" %}</a>
     {% with course=request.course %}
-      {% include "header.html" %}
+      {% include "header.html"|microsite_template_path %}
```

Comment:
@mattdrayer make tag `## mako` is missing in our microsite repo that is why there is a breakage in header section of wiki. I think we need to add up this in order to header loading. 

@mattdrayer @felipemontoya  It seems me that microsite css is not rendering in wiki. we need to override the microsite css.  I just tested it after adding the `{% microsite_css_overrides_file %}` in `main_django.html` and styles are rendering OK.